crates/uv-cli/src/lib.rs

```diff
@@ -2101,18 +2121,12 @@ pub struct RunArgs {
     #[arg(long, alias = "no_workspace", conflicts_with = "package")]
     pub no_project: bool,
 
-    /// The Python interpreter to use to build the run environment.
-    ///
-    /// By default, uv uses the virtual environment in the current working directory or any parent
-    /// directory, falling back to searching for a Python executable in `PATH`. The `--python`
-    /// option allows you to specify a different interpreter.
+    /// The Python interpreter to use for the run environment.
     ///
-    /// Supported formats:
+    /// If the interpreter request is satisfied by a discovered environment, a
+    /// new one will not be created.
```

Comment:
Could we maybe rephrase as like... "it will be reused" rather than "not created"?